src/dialects/abstract/query-interface-typescript.ts

```diff
@@ -12,16 +14,99 @@ export class AbstractQueryInterfaceTypeScript {
   }
 
   /**
-   * Creates a new database schema.
+   * Create a new database schema.
    *
    * **Note:** this is a schema in the [postgres sense of the word](http://www.postgresql.org/docs/9.1/static/ddl-schemas.html),
-   * not a database table. In mysql and sqlite, this command will do nothing.
+   * not a database table. In mysql and mariadb, this command will instead create a database.
    *
-   * @param schema
-   * @param options
+   * @see
+   * {@link Model.schema}
+   *
+   * @param schema Name of the schema
+   * @param [options={}]
+   * @param [options.collate=null]
+   * @param [options.charset=null]
+    *
+   * @returns
    */
   async createSchema(schema: string, options?: CreateSchemaOptions): Promise<void> {
     const sql = this.queryGenerator.createSchemaQuery(schema, options);
     await this.sequelize.queryRaw(sql, options);
   }
+
+  /**
+   * Drop a single schema
+   *
+   * **Note:** this is a schema in the [postgres sense of the word](http://www.postgresql.org/docs/9.1/static/ddl-schemas.html),
+   * not a database table. In mysql and sqlite, this drop a table matching the schema name
```

Comment:
```suggestion
   * not a database table. In mysql and mariadb, this drops a database matching the schema name
```

Reply:
I'm not 100% sure on this and my following suggestions but it follows from the logic that databases and schemas are interchangable for those two. We might want to add additional information on the behaviour on sqlite since that might show/drop all tables, although we're not testing any of these functions on sqlite. (we might want to do that just to make sure though)

Reply:
We should copy the updated jsdoc from here to `sequelize.js` as well so that they're the same (or reference these in these with the `@see` option)

Reply:
Sounds good, though I was going to remove these methods from `Sequelize` after this PR gets merged. Is it worth copying over?

Reply:
Ah, I thought we would just deprecate them in v7